fineract-provider/src/main/resources/db/changelog/tenant/parts/0002_initial_data.xml

```diff
@@ -1916,7 +1916,7 @@
             <column name="code" value="CRC"/>
             <column name="decimal_places" valueNumeric="2"/>
             <column name="currency_multiplesof"/>
-            <column name="display_symbol" value="₡"/>
+            <column name="display_symbol"/>
```

Comment:
Let's not change anything in the existing changelog files. If you would like to remove the display_symbol for these currencies, let's add a new changelog file with the changes needed.
Btw, any reason you decided to remove these?